test/python/test_identifiers.py

```diff
@@ -24,6 +24,8 @@
                     QuantumRegister, QuantumProgram)
 from .common import QiskitTestCase
 
+available_backends = QuantumProgram().available_backends()
```

Comment:
Since #376, we are moving away from relying on QuantumProgram for these kinds of things, and use the `wrapper.available_backends()` instead. 

However, probably a better way to check a local simulator is installed is by instantiating it and using try/except. See the tests in [test_simulator_interfaces](https://github.com/QISKit/qiskit-sdk-py/blob/master/test/python/test_simulator_interfaces.py) and [test_qasm_simulator_projectq](https://github.com/QISKit/qiskit-sdk-py/blob/master/test/python/test_qasm_simulator_projectq.py) and [test_qasm_simulator_cpp](https://github.com/QISKit/qiskit-sdk-py/blob/master/test/python/test_qasm_simulator_cpp.py).

Reply:
Fixed by @ajavadia in https://github.com/QISKit/qiskit-sdk-py/pull/395/commits/a73b940a60c427c803b8219d780b5038ae0959e5